png_benchmark.py

```diff
@@ -54,7 +55,11 @@
 HEADER_LINE = ",".join(HEADERS) + "\n"
 
 
-def update_min_max(min, max, current_value):
+def update_min_max(
+        min_value: Union[int, float],
+        max_value: Union[int, float],
+        current_value: Union[int, float],
+) -> Tuple[Union[int, float], Union[int, float]]:
```

Comment:
Use this notation:
```py
var1: "int | float"
var2: "tuple[int | float, int | float]"

# The intended output would be something like this:
var3: "tuple[int, int] | tuple[float, float]"
# but it's not guaranteed because the inputs could be a combination of int and float
```